config/locales/server.en.yml

```diff
@@ -1067,6 +1067,7 @@ en:
     search_recent_posts_size: "How many recent posts to keep in the index"
     log_search_queries: "Log search queries performed by users"
     search_query_log_max_size: "Maximum amount of search queries to keep"
+    search_query_log_max_retention_days: "Maximum amount of time to keep search queries, in days. Default is 3 years (1095 days)."
```

Comment:
default should be 6 months, I do not see why we need to hoard 3 years here.

Reply:
The admin dashboard is trying to do year-over-year analytics on search queries; should search logs be limited to month/quarter scope?

Reply:
largest interval there is a year, so just keep 1 year max I guess.